reggie/configs/data/west_virginia.yaml

```diff
@@ -93,72 +121,245 @@ column_names:
+  - int
+  - int
+  - int
+  - int
+  - int
+  - int
+  - int
+  - int
+  - int
+  - int
+  - int
+
+# Voter history
+##############################
+
+# id_voter|id_election|dt_election|Election_Name|cd_election_type|cd_election_cat|fl_absentee|fl_challenged|fl_early_voting|fl_regular
+hist_columns:
+  - id_voter
+  - id_election
+  - dt_election
+  - Election_Name
+  - cd_election_type
+  - cd_election_cat
+  - fl_absentee
+  - fl_challenged
+  - fl_early_voting
+  - fl_regular
+
+hist_columns_types:
+  id_voter: text
+  id_election: text
+  dt_election: date
+  Election_Name: text
+  cd_election_type: text
+  cd_election_cat: text
+  fl_absentee: text
+  fl_challenged: text
+  fl_early_voting: text
+  fl_regular: text
+
+# Voter history and other fields
+##############################
+generated_columns:
+  all_history: text[]
+  sparse_history: int[]
+  # absentee, early-voting, regular
+  votetype_history: text[]
+  # Challenged (boolean)
+  challenged_history: integer[]
+ordered_generated_columns:
+  - all_history
+  - sparse_history
+  - votetype_history
+  - challenged_history
+
+# Custom configuration
+# (legacy, unsure if needed yet)
+##############################
 status_codes:
-  active: Active
-  inactive: Inactive
+  active:
+    - Active
+  inactive:
+    - Inactive
 
 gender_codes:
-  Male:
+  male:
     - M
     - m
-  Female:
+  female:
     - F
     - f
-  Unknown:
+  unknown:
     - N
+    - n
     - nan
-    -
-
-race_codes:
-  Not_Provided: NP
-
-party_codes:
-  no_party_affiliation:
-    - No Party Affiliation
-    - Unaffiliated
-  democratic_party: Democrat
-  republican_party: Republican
-  green_party: Green
-  libertarian_party: Libertarian
-  mountain_party: Mountain
-  independent:
-    - Independent
-    - Independent Party
-  other: Other
-
-num_party_codes: 230
+    - NaN
+    - ""
+    - " "
+
+# Party codes are very variable, so done in code
+
+locales_counties:
+  barbour: Barbour
+  berkeley: Berkeley
+  hancock: Hancock
+  jefferson: Jefferson
+  boone: Boone
+  braxton: Braxton
+  brooke: Brooke
+  cabell: Cabell
+  calhoun: Calhoun
+  clay: Clay
+  doddridge: Doddridge
+  fayette: Fayette
+  gilmer: Gilmer
+  grant: Grant
+  greenbrier: Greenbrier
+  hampshire: Hampshire
+  hardy: Hardy
+  harrison: Harrison
+  jackson: Jackson
+  marion: Marion
+  kanawha: Kanawha
+  lewis: Lewis
+  lincoln: Lincoln
+  logan: Logan
+  mason: Mason
+  monongalia: Monongalia
+  marshall: Marshall
+  mcdowell: McDowell
+  mercer: Mercer
+  mineral: Mineral
+  mingo: Mingo
+  morgan: Morgon
+  monroe: Monroe
+  nicholas: Nicholas
+  raleigh: Raleigh
+  ohio: Ohio
+  pendleton: Pendleton
+  pleasants: Pleasants
+  pocahontas: Pocahontas
+  preston: Preston
+  putnam: Putnam
+  randolph: Randolph
+  ritchie: Ritchie
+  roane: Roana
+  summers: Summers
+  taylor: Taylor
+  tucker: Tucker
+  tyler: Tyler
+  upshur: Upshur
+  wayne: Wayne
+  webster: Webster
+  wetzel: Wetzel
+  wirt: Wirt
+  wood: Wood
+  wyoming: Wyoming

```

Comment:
In addition to the yaml, you will also eventually need to add to reggie a copy of what you will create for the frontend "frontend/src/data/counties/wv.json" file. That goes in reggie/configs/primary_locale_names/west_virginia.json